Sources/TokamakCore/MountedViews/MountedApp.swift

```diff
@@ -34,49 +34,43 @@ final class MountedApp<R: Renderer>: MountedCompositeElement<R> {
     mountedChildren.forEach { $0.unmount(with: reconciler) }
   }
 
-  private func mountChild<S: Scene>(_ childBody: S) -> MountedElement<R> {
-    let mountedScene: MountedScene<R> = childBody.makeMountedView(parentTarget,
-                                                                  environmentValues)
+  private func mountChild(_ childBody: _AnyScene) -> MountedElement<R> {
+    let mountedScene: MountedScene<R> = childBody.makeMountedScene(parentTarget, environmentValues)
     if let title = mountedScene.title {
       // swiftlint:disable force_cast
       (app.type as! _TitledApp.Type)._setTitle(title)
     }
-    return mountedScene.body
+    return mountedScene
   }
 
   override func update(with reconciler: StackReconciler<R>) {
     let element = reconciler.render(mountedApp: self)
     reconciler.reconcile(
       self,
       with: element,
-      getElementType: { ($0 as? _AnyScene)?.type ?? type(of: $0) },
-      updateChild: { $0.scene = _AnyScene(element) },
+      getElementType: { $0.type },
+      updateChild: {
+        $0.environmentValues = environmentValues
```

Comment:
This is where environment was not passed from a parent to its children previously.